packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -78,20 +78,20 @@
   color: $ms-color-themePrimary;
 }
 
-.root:hover {
-  border-color: $ms-color-themeDarker;
+.root:hover:not(.rootIsDisabled):not(.rootIsActive) {
```

Comment:
huh. Can you avoid these `:not` qualifiers? It seems that TextField uses `pointer-events: none` to avoid hover effects.